contrib/completion/bash/docker

```diff
@@ -3634,6 +3635,25 @@ _docker_plugin_set() {
 	esac
 }
 
+_docker_plugin_upgrade() {
+	case "$cur" in
+		-*)
+			COMPREPLY=( $( compgen -W "--disable-content-trust --grant-all-permissions --help --skip-remote-check" -- "$cur" ) )
+			;;
+		*)
+			local counter=$(__docker_pos_first_nonflag '--timeout')
```

Comment:
@mlaventure `--timeout` is nonsense here (though it doesn't harm), I'll fix it in a moment.